fileio/simpleio.go

```diff
@@ -88,7 +89,32 @@ func CatchErrThrowEOF(err error) {
 // Close will return an error if it has already been called.
 func (reader *SimpleReader) Close() {
 	if reader != nil {
-		err := reader.file.Close()
-		common.ExitIfError(err)
+		common.ExitIfError(reader.close())
 	}
 }
+
+// StringToIntSlice will process a column data separated by commas, convert the slice into a slice of type int.
+func StringToIntSlice(column string) []int {
+	work := strings.Split(column, ",")
+	var answer []int = make([]int, len(work))
+	for i := 0; i < len(work)-1; i++ {
```

Comment:
Why does this ignore the last element in `work`? If this is to deal with data types with a trailing comma it might be good to make a note in the comment, otherwise someone may try to use it and have the end of their data cut off.

Reply:
Maybe we could check to see if the last character is a comma and then handle both cases?

Reply:
Yeah that was why I originally had it as a small private function in PSL, since it was specific to some of UCSC's file type anomalies. I have a better idea as a proposed change.